src/main/java/io/openshift/launchpad/ui/booster/ProjectInfoStep.java

```diff
@@ -63,6 +65,7 @@
 public class ProjectInfoStep implements UIWizardStep
 {
    private static final Logger logger = Logger.getLogger(ProjectInfoStep.class.getName());
+   private static final String RETRY_STEP = "RETRY_STEP";
```

Comment:
It should be named `Retry-Step` if you intend to pass it as a header parameter named `X-Retry-Step`. In this case it will look for a `X-RETRY_STEP`, which is fine too, just making sure it is clear enough

Reply:
right